.github/workflows/create-release.yml

```diff
@@ -58,12 +59,19 @@ jobs:
           git config --global user.email '${{ steps.get-user-id.outputs.user-id }}+${{ steps.generate_token.outputs.app-slug }}@users.noreply.github.com'
           ./.tools/multimod tag --module-set-name liatrio-otel --commit-hash '${{ github.sha }}' --print-tags
           git push --tags
-
+      - name: Get changelog
+        if: steps.get_next_version.outputs.hasNextVersion == 'true'
```

Comment:
I am not sure we need this commit. This whole workflow will only trigger if the head commit starts with `Prepare liatrio-otel for version` which can only happen if we detect a new version. Also we can remove `get-next-version` step entirely and get the next semvar from `github-tag-action`

Reply:
Can you clarify what you mean by the entire commit? Removed the `get-next-version` step and used the tag from `steps.get_tag.outputs.new_tag`

Reply:
Yeah 'commit' was a typo. I was referring to the action.